File: testers/combined/java/generated-and-checked-in/api/combined/api/api/EmployeesApi.java
package combined.api.api;

import combined.api.model.Employee;
import io.smallrye.mutiny.Uni;
import java.util.List;
import java.util.Optional;

public interface EmployeesApi {
  /** Get employee by ID */
  Uni<Employee> getEmployee(Integer employeeId);

  /** List all employees */
  Uni<List<Employee>> listEmployees(
  
    /** Filter by active status */
    Optional<Boolean> isActive
  );
}

File: testers/combined/kotlin/generated-and-checked-in/api/combined/api/api/EmployeesApi.kt
package combined.api.api

import combined.api.model.Employee
import io.smallrye.mutiny.Uni
import kotlin.collections.List

interface EmployeesApi {
  /** Get employee by ID */
  abstract fun getEmployee(employeeId: Int): Uni<Employee>

  /** List all employees */
  abstract fun listEmployees(
    /** Filter by active status */
    isActive: kotlin.Boolean?
  ): Uni<List<Employee>>
}